internal/store/store.go

```diff
@@ -200,6 +200,23 @@ func NewUnauthenticated(c Config) (*Store, error) {
 
 	return s, nil
 }
+func NewStaticPath(c Config, baseURL string) (*Store, error) {
```

Comment:
related to my previous comment, `baseURL` belongs in this package, it will also be nice if we can parameterize this via the config object but not necessary right now